synapse/config/server.py

```diff
@@ -505,6 +505,9 @@ class LimitRemoteRoomsConfig(object):
             "cleanup_extremities_with_dummy_events", True
         )
 
+        # The number of forward extremities in a room needed to send a dummy event.
+        self.dummy_event_threshold = config.get("dummy_event_threshold", 10)
+
```

Comment:
please can this have some documentation

Reply:
Fixed in https://github.com/matrix-org/synapse/pull/7422/commits/8cab845d783543f7d057f6a6be0a5d35bd268489